### DISCUSSION TITLE
can not see the type information when hovering over the `vector`

### DISCUSSION THREAD

**Discussion by mzwsw-wjm** [microsoft/vscode-cpptools#11068] [Q&A]
Hello everyone, i'm new to Vscode and I meet some question :
I'm following the helloworld tutorial : https://code.visualstudio.com/docs/cpp/config-mingw
I have installed the corresponding plugin : 
![image](https://github.com/microsoft/vscode-cpptools/assets/53400616/4fa94f2b-7202-4544-97ce-5de378ce70ef)
 I have already installed the necessary compilers : 
![image](https://github.com/microsoft/vscode-cpptools/assets/53400616/b0dd68a8-8d7c-4209-93e8-060524407ee0)
The OS i use is Windows 10.
But when hover over the vector, i can not see the type information : 
![image](https://github.com/microsoft/vscode-cpptools/assets/53400616/eef7afa8-0df2-49d0-b8ef-f1f210640957)
The right form is as following ： 
![image](https://github.com/microsoft/vscode-cpptools/assets/53400616/000a9449-57ed-40e0-89fd-271c2ff57fb1)

And i cannot find the play button in the top right corner of the editor : 
![image](https://github.com/microsoft/vscode-cpptools/assets/53400616/6015e237-4036-4abe-800b-09efff1f2a99)

I want to know why and what can i do to fix it. Thanks!

**Comment by michelleangela** [comment]
@mzwsw-wjm 
while you have your C++ source file open, could you run the command `C/C++: Log Diagnostics` from the VS Code command palette and share those logs here

Could you also get logs from the language server (see https://code.visualstudio.com/docs/cpp/enable-logging-cpp#_enable-logging-for-the-language-server)?
